rstblog/config.py

```diff
@@ -39,7 +39,7 @@ def list_entries(self, key):
         for layer in self.stack:
             for key, value in layer.iteritems():
                 if key.startswith(prefix):
-                    rv[key] = value
+                    rv[key[len(prefix):]] = value
```

Comment:
Right, I don't know if it would down the road. But like I said AFAICT it's only being used in one place now, and this fixes that use case.